### DISCUSSION TITLE
Passkeys support?

### DISCUSSION THREAD

**Discussion by x86y** [FiloSottile/age#567] [General]
I couldn't find if this has been asked or discussed previously, but I think using hardware as an age secret input would be very useful. Passkeys are getting almost universally adopted. I'm not sure about the implementation details for this, as it may very well be out of age's scope, but I'm interested to hear your thoughts about it!

**Comment by cipriancraciun** [comment]
Based on my limited understanding on how Passkeys (or WebAuthn) works, I would say it's not quite possible, because that standard involves signing (in hardware), not decrypting.  Also, I don't think key management would actually work (especially with non-persistent keys).
